cmake/macros/AddCrashpad.cmake

```diff
@@ -23,12 +23,12 @@ macro(add_crashpad)
     set(CMAKE_BACKTRACE_TOKEN $ENV{CMAKE_BACKTRACE_TOKEN})
   endif()
 
-  if ((WIN32 OR APPLE) AND USE_CRASHPAD)
+  if (USE_CRASHPAD)
     get_property(CRASHPAD_CHECKED GLOBAL PROPERTY CHECKED_FOR_CRASHPAD_ONCE)
     if (NOT CRASHPAD_CHECKED)
 
       add_dependency_external_projects(crashpad)
-      find_package(crashpad REQUIRED)
+      find_package(Crashpad REQUIRED)
```

Comment:
For the cmake warning: Is there any reason you changed "crashpad" to "Crashpad"? `Fetch is getting "crashpad" https://chromium.googlesource.com/chromium/tools/depot_tools.git/+/refs/heads/master/fetch_configs/crashpad.py and I don't see any "Crashpad" in the files for crashpad either.